queries/djot/highlights.scm

```diff
@@ -85,106 +90,168 @@
 ] @markup.list
 
 (list_marker_task
-  (unchecked) @constant.builtin) @markup.list.unchecked
+  (unchecked)) @markup.list.unchecked
 
 (list_marker_task
-  (checked) @constant.builtin) @markup.list.checked
+  (checked)) @markup.list.checked
+
+; Colorize `x` in `[x]`
+((checked) @constant.builtin
+  (#offset! @constant.builtin 0 1 0 -1))
 
 [
   (ellipsis)
   (en_dash)
   (em_dash)
-  (straight_quote)
 ] @string.special
 
+(list_item
+  (term) @type.definition)
+
+(quotation_marks) @string.special
+
+; Conceal { and } but leave " and '
+((quotation_marks) @string.special
+  (#eq? @string.special "{\"")
+  (#offset! @string.special 0 0 0 -1)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#eq? @string.special "\"}")
+  (#offset! @string.special 0 1 0 0)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#eq? @string.special "{'")
+  (#offset! @string.special 0 0 0 -1)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#eq? @string.special "'}")
+  (#offset! @string.special 0 1 0 0)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#any-of? @string.special "\\\"" "\\'")
+  (#offset! @string.special 0 0 0 -1)
+  (#set! conceal ""))
+
+(hard_line_break) @string.escape
+
+(backslash_escape) @string.escape
+
+; Only conceal \ but leave escaped character.
+((backslash_escape) @string.escape
+  (#offset! @string.escape 0 0 0 -1)
+  (#set! conceal ""))
+
 (frontmatter_marker) @punctuation.delimiter
 
 (emphasis) @markup.italic
 
 (strong) @markup.strong
 
-(emphasis
-  (emphasis_begin) @punctuation.delimiter)
+(symbol) @string.special.symbol
+
+(insert) @markup.underline
 
-(emphasis
-  (emphasis_end) @punctuation.delimiter)
+(delete) @markup.strikethrough
 
-(strong
-  (strong_begin) @punctuation.delimiter)
+(highlighted) @string.special
 
-(strong
-  (strong_end) @punctuation.delimiter)
+(superscript) @string.special
 
+(subscript) @string.special
+
+; We need to target tokens specifically because `{=` etc can exist as fallback symbols in
+; regular text, which we don't want to highlight or conceal.
 (highlighted
   [
     "{="
     "=}"
-  ] @punctuation.delimiter)
+  ] @punctuation.delimiter
+  (#set! conceal ""))
 
 (insert
   [
     "{+"
     "+}"
-  ] @punctuation.delimiter)
+  ] @punctuation.delimiter
+  (#set! conceal ""))
 
 (delete
   [
     "{-"
     "-}"
-  ] @punctuation.delimiter)
+  ] @punctuation.delimiter
+  (#set! conceal ""))
```

Comment:
These look like they can be grouped into one

```suggestion
([
    "{="
    "=}"
    "{+"
    "+}"
    "{-"
    "-}"
] @punctuation.delimiter
  (#set! conceal ""))
```

Reply:
So are the ones below it seems

Reply:
I don't think so, it's important to target child nodes specifically (e.g. `(delete "{-")` but not `{-` globally because there exists an anonymous node that captures `{-` that would den get wrongly highlighted and concealed.

Reply:
As you are the writer of the language, is there any other `{-` node?  From what I can see it's only here and symbol_fallback, which doesn't matter

Reply:
Yes I am, and it's symbol_fallback I'm referring to.

I also didn't think it shouldn't matter, but when I use the global version it does get highlighted and concealed and I can only assume that symbol_fallback is the culprit.